pixeltable/catalog/table.py

```diff
@@ -983,6 +983,7 @@ def add_embedding_index(
         string_embed: pxt.Function | None = None,
         image_embed: pxt.Function | None = None,
         metric: Literal['cosine', 'ip', 'l2'] = 'cosine',
+        precision: Literal['16bit', '32bit'] = '16bit',
```

Comment:
These are unusual names. The 'standard' names (that I've seen most often) are `float32`, `float16`, and `bfloat16`.  [The latter has a different ratio of mantissa bits to exponent bits (8:8) than the IEEE float16 (12:4).] I'd suggest we should use either `float16` or `bfloat16` depending on which is the actual format pgvector uses.